### PR TITLE
Detox Build Error Android

### LINKED ISSUE #126: Detox build failing Android

### PR DESCRIPTION
**Pull request by chirag-singhal**
Fixes #126 

Also, made changes to pass existing android tests.
Screenshot from 2020-07-06 19-02-03

### CODE REVIEW

File: packages/RNTester/.detoxrc.json [hunk 1]
```diff
@@ -4,15 +4,15 @@
   "configurations": {
     "android.emu.debug": {
       "binaryPath": "android/app/build/outputs/apk/debug/app-debug.apk",
-      "build": "cd android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ..",
+      "build": "cd android && ./gradlew assembleDebug app:assembleAndroidTest -DtestBuildType=debug && cd ..",
       "type": "android.emulator",
       "device": {
-        "avdName": "testAVD"
+        "avdName": "Pixel_API_28"
```
Comment (stealthanthrax) [inline packages/RNTester/.detoxrc.json:10]: All looks good, just one concern. Will this work if the avdName is not 'Pixel_API_28' ?

Comment (manyaagarwal) [reply packages/RNTester/.detoxrc.json:10]: Yes, we'll have to change this to whatever our device is called I'm guessing.

Comment (stealthanthrax) [reply packages/RNTester/.detoxrc.json:10]: If that's the case, @chirag-singhal , can you please also add it in the readme ?
> Yes, we'll have to change this to whatever our device is called I'm guessing.

Comment (chirag-singhal) [reply packages/RNTester/.detoxrc.json:10]: @stealthanthrax there is a wiki link in readme which tells detailed steps to run tests on android. Should it be included in Readme as well?

Comment (stealthanthrax) [reply packages/RNTester/.detoxrc.json:10]: @chirag-singhal , wasn't aware of it. I would suggest that you mention it in the readme too 😅

**Pull request merged**